src/View/Helper/FormHelper.php

```diff
@@ -1597,7 +1597,7 @@ public function checkbox(string $fieldName, array $options = []): array|string
      *   the radio label will be 'empty'. Set this option to a string to control the label value.
      *
      * @param string $fieldName Name of a field, like this "modelname.fieldname"
-     * @param iterable $options Radio button options array.
+     * @param iterable|array $options Radio button options array.
```

Comment:
This is incorrect, `iterable` is already `array|Traversable` https://www.php.net/manual/en/language.types.iterable.php